.github/workflows/release-rancher-desktop.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Release Rancher Dashboard - Rancher Desktop Embed
+
+on:
+  push:
+    tags:
+        - 'v**-desktop*'
+
+env:
+  OUTPUT_DIR: dist
+  RELEASE_DIR: release
+  ARTIFACT_NAME: rancher-dashboard-desktop-embed
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+    
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '12.x'
+    
+    - name: Install & Build
+      run: |
+        mkdir ${{ env.RELEASE_DIR }}
```

Comment:
I think, not sure, it's normally better practise to pull these sections out in to their own file. There's some example bash files in `./scripts` that drone uses. The file could live in `.github/workflows/scripts`

Reply:
I've done it in other areas, for instance in [our build script for Steve](https://github.com/rancher-sandbox/steve/blob/main/bin/build.sh). I think that it would make sense to break out into separate build script if we added any more complexity.